README.md

```diff
@@ -3,7 +3,31 @@
 More information on this service can be found on Confluence:
 
 * https://confluence.ons.gov.uk/display/SDC/SDS
+---
+## Dockerized
+The docker-compose will launch the SDS application, two storgae emulators(firebase and bucket) and two cloud functions. The SDS application will also support hot reloading within the `/src/app` directory.
 
+- You will need to create a new file called `mock_google_app_key.json` within the `/devtools` directory and copy the contents from this fake service acount found here [Mock service account](https://github.com/firebase/firebase-admin-python/blob/master/tests/data/service_account.json).
```

Comment:
Are you sure you need `mock_google_app_key.json`. Last time I managed to ditch this file by setting `FIRESTORE_EMULATOR_HOST`

Reply:
You can test this by removing the GOOGLE_APPLICATION_KEY from the main api in the docker-compose. The code will complain that there are no credentials set.